### DISCUSSION TITLE
Tailwind Elements Stable v1.0.0.  - a free, open-source UI Kit with 500+ components integrated with Laravel - is out.

### DISCUSSION THREAD

**Discussion by MaciejZimoch** [laravel/framework#48628] [Show and tell]
Tailwind Elements Stable v1.0.0.  - a free, open-source UI Kit with 500+ components integrated with Laravel - is out. 
And we couldn't be prouder, so give it a go [here:](https://tailwind-elements.com/docs/standard/integrations/react-integration/) 

It will change Your life (or at least how smooth & painless web development is) ;)

It's stable & reliable, comes with over 500 UI Components, easy theming & customisation, dark mode, Figma design system AND free hosting :) . 

Best known for offering a clean and elegant syntax that is designed to make web development tasks straightforward and quick, Laravel integration in Tailwind Elements is even more elegant than it has ever been, while still aiding rapid application development.
 
For all JS aficionados: By default all components have autoinit which means they are initialised by data attributes. But if you want to make init by JS - there is also possibility to do that. 

Check out how to do it and more about Tailwind Elements [here](https://tailwind-elements.com/docs/standard/integrations/react-integration/) 

Even though TE is already a fully functional UI Kit, it's still in its infancy really - we have great plans and a long and exciting road ahead of us. So whatever input (however critical) You might have, please share it with us here or on any of our channels on which we will be more than happy to host you: 
[Discord](https://discord.gg/ZkTVKaJC) 
[GitHub ](https://github.com/mdbootstrap/Tailwind-Elements)
[X ](https://twitter.com/TailwindElement)

**Comment by Sophist-UK** [comment]
Is there a Livewire version of this?
